### PR TITLE
Updating readme links for CTP5

### PR DESCRIPTION
**Pull request by jmarolf**
This adds additional info for contributors who want to write analyzers on top of Visual Studio 2015 CTP5.  Just added a new section with the updated links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +36,19 @@ to get the Visual Studio project templates.
 to get a [Syntax Visualizer tool window](https://github.com/dotnet/roslyn/wiki/Syntax%20Visualizer) 
 to help explore the syntax trees you'll be analyzing.
 
+To get started on **Visual Studio 2015 CTP5**:
+
+1. Set up a box with Visual Studio 2015 CPT5. Either 
+[install  Visual Studio 2015 Preview](http://go.microsoft.com/fwlink/?LinkId=400496),
```
Comment (dpoeschl) [inline README.md]: Should this read "install Visual Studio 2015 CTP5"?

File: README.md [hunk 2]
```diff
@@ -34,6 +36,19 @@ to get the Visual Studio project templates.
 to get a [Syntax Visualizer tool window](https://github.com/dotnet/roslyn/wiki/Syntax%20Visualizer) 
 to help explore the syntax trees you'll be analyzing.
 
+To get started on **Visual Studio 2015 CTP5**:
+
+1. Set up a box with Visual Studio 2015 CPT5. Either
```
Comment (rchande) [inline README.md]: "CPT5" -> "CTP5"

File: README.md (hunk 1 above)
Comment (jmarolf) [inline README.md]: Yep, lets try this again...

File: README.md [hunk 3]
```diff
@@ -34,6 +36,19 @@ to get the Visual Studio project templates.
 to get a [Syntax Visualizer tool window](https://github.com/dotnet/roslyn/wiki/Syntax%20Visualizer) 
 to help explore the syntax trees you'll be analyzing.
 
+To get started on **Visual Studio 2015 CTP 5**:
+
+1. Set up a box with Visual Studio 2015 CPT 5. Either
```
Comment (rchande) [inline README.md]: It still says "CPT" :frog:

**Comment by dpoeschl** [comment]
Looks great. :+1:

**Comment by rchande** [comment]
:+1:

**Pull request merged**